keras/backend/mxnet_backend.py

```diff
@@ -5413,7 +5500,8 @@ def _adjust_module(self, inputs, phase):
 
             if not self._module.binded:
```

Comment:
*nit, bind ==> bound (not binded)

Reply:
Can't help it is defined in MXNet Module API.